assets/js/src/form_editor/templates/components/template_box.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import MailPoet from 'mailpoet';
+import Heading from 'common/typography/heading/heading';
+import Button from 'common/button/button';
+import { TemplateType } from 'form_editor/templates/store/types';
+
+type Props = {
+  template: TemplateType,
+  onSelect: (id: string) => any,
+}
+const TemplateBox = ({ template, onSelect }: Props) => (
+  <div className="mailpoet-template mailpoet-form-template" data-automation-id="select_template_box">
+    <div className="mailpoet-template-thumbnail">
+      <img src={template.thumbnail} alt={template.name} />
```

Comment:
Should we specify the dimension here so that the browser can calculate the space before loading the image? Also, we could add an argument `loading="lazy"` for more optimal loading...